finance-api/user-guides/fetch-account-balance.md

```diff
@@ -0,0 +1,52 @@
+Fetch account balance
+===
+Using the Finance API, you can fetch the balance that is available in the merchant's Zettle preliminary or liquid account.
```

Comment:
I think we can drop "that is available" to simplify.
Balance is always available 

```suggestion
Use the Finance API to fetch the account balance of the the merchant's preliminary or liquid account.
```